Doppler.HtmlEditorApi/DopplerHtmlDocument.cs

```diff
@@ -51,6 +53,15 @@ public string GetDopplerContent()
     public string GetHeadContent()
         => _headNode?.InnerHtml;
 
+    public IEnumerable<int> GetFieldsId()
+    {
+        return FIELD_ID_TAG_REGEX.Matches(_contentNode.InnerHtml)
+        .Select(x => x.Groups[1].Value)
+        .Select(int.Parse)
+        .Distinct();
+    }
```

Comment:
What do you think about using the short version?

```suggestion
    public IEnumerable<int> GetFieldsId()
        => FIELD_ID_TAG_REGEX.Matches(_contentNode.InnerHtml)
            .Select(x => x.Groups[1].Value)
            .Select(int.Parse)
            .Distinct();
```

(I have no a strong opinion on this)